src/main/java/dev/julialoz/game/javarushgame/GameBuilder.java

```diff
@@ -0,0 +1,232 @@
+package dev.julialoz.game.javarushgame;
+
+import dev.julialoz.game.javarushgame.model.Enemy;
+import dev.julialoz.game.javarushgame.model.Location;
+import dev.julialoz.game.javarushgame.model.LocationName;
+import dev.julialoz.game.javarushgame.model.User;
+import dev.julialoz.game.javarushgame.model.item.*;
+import dev.julialoz.game.javarushgame.model.npc.Answer;
+import dev.julialoz.game.javarushgame.model.npc.Message;
+import dev.julialoz.game.javarushgame.model.npc.Npc;
+import dev.julialoz.game.javarushgame.model.quest.CollectingQuest;
+import dev.julialoz.game.javarushgame.model.quest.Quest;
+
+import java.util.ArrayList;
+import java.util.List;
+
+// items
+public class GameBuilder {
+    private List<Item> items = new ArrayList<>();
+    private List<Npc> npcs = new ArrayList<>();
+    ;
+    private List<Enemy> enemys = new ArrayList<>();
+    ;
+    private List<Location> locations;
+    private List<Message> messages = new ArrayList<>();
+    private List<Answer> answers = new ArrayList<>();
+    private List<Quest> quests = new ArrayList<>();
+    ;
+
+    public GameBuilder() {
+        items = generateItems();
+        quests = generateQuests();
+        answers = generateAnswers();
+        messages = generateMessages();
+        npcs = generateNpcs();
+        locations = generateLocations();
+    }
+
+    private List<Quest> generateQuests() {
+        List<Quest> quests = new ArrayList<>();
+        // ladybug quest
+        quests.add(CollectingQuest.builder()
+                .id(1l)
+                .questHolderId(1L) // ladybug id
+                .description("Find some water in the swamp")
+                .requiredItem(4L)
+                .build());
+        return quests;
```

Comment:
```suggestion
        quests.add(createQuest(1L,1L,"Text", 4L));
```
 я бы для каждого билдера внутри, создал бы метод, тогда размер кода значительно сократился бы